distributions/JavaScript/INSTRUCTIONS

```diff
@@ -0,0 +1,27 @@
+1. Install Enscriptem
+    # Get the emsdk repo
+    git clone https://github.com/emscripten-core/emsdk.git
+
+    # Enter that directory
+    cd emsdk
+
+    # Fetch the latest version of the emsdk (not needed the first time you clone)
+    git pull
+
+    # Download and install the latest SDK tools.
+    ./emsdk install latest
```

Comment:
If we rewrite the documenation lets do it correctly.
I've tried on fresh docker with ubuntu.

Up to this point everything worked fine, until I've seen:

```
root@48d646fe631a:/library/distributions/JavaScript/emsdk# ./emsdk install latest 
/usr/bin/env: 'python': No such file or directory
```

Please mention all necessary requirements needed to work with this code.
Make sure you tested it on clean OS / docker image.

Reply:
I am testing it on a clean Ubuntu VM and the only prerequisites are Python and cmake, they will be addressed in the instruction.

Reply:
What about `git` and `wget` ?